docs/source/pillars/data_management.md

```diff
@@ -228,159 +223,6 @@ The ability to query and browse the data within an environment at various levels
   - Recommended
 ```
 
-(standard_capability_information_security)=
-
-## Information security
-
-The ability of the {ref}`TRE operator <infrastructure_roles>` to protect against the unauthorised use of information, especially electronic data.
-
-Measures taken to ensure information security can be further categorised into:
-
-- {ref}`vulnerability management <vulnerability-management>`: applying security updates or fixes for identified vulnerabilities
-- {ref}`security testing <security-testing>`: proactive penetration testing of a deployed system
-- {ref}`encryption <encryption>`: ensuring that data is protected even if the TRE is compromised
-- {ref}`physical security <physical-security>`: restricting TRE access to known secure locations
-
-A TRE conforming to the SATRE standard should enact broadly similar measures to protect against the unauthorised use of information, especially electronic data.
-These measures include vulnerability management of TRE infrastructure (whether physical or virtual/cloud-based), carrying out compliance checks and security tests of the TRE, common approaches to data encryption, and (where appropriate) physical security measures to prevent unauthorised access to the TRE.
-
-(vulnerability-management)=
-
-### Vulnerability Management
-
-The ability of the {ref}`TRE operator <infrastructure_roles>` to identify, assess, report on, manage and remediate technical vulnerabilities across endpoints, workloads, and systems.
-
-```{list-table}
-:header-rows: 1
-:name: tab-vulnerability-management
-
-* -
-  - Statement
-  - Guidance
-  - Importance
-
-* - 3.5.1.
-  - You must have a process in place for applying security updates to all software that forms part of the TRE infrastructure.
-  - This includes any software used for remote desktop portals, databases, webapps, creating and destroying compute infrastructure, configuration management, or software used for monitoring the TRE.
-  - Mandatory
-* - 3.5.2.
-  - You must run regular anti-virus/malware scans on all TRE systems where infection could be a problem.
-  - Virus and malware scans will help identify malicious code which may compromise the security, or correct operation, of the TRE.
-  - Mandatory
-* - 3.5.3.
-  - Your TRE should adhere to one or more external security standards.
-  - These should be stated to all stakeholders in advance of any data being brought in to the TRE.
-  - Recommended
```

Comment:
I haven't included this in the new computing technology information security section, as I don't think it fits in.

If we want to recommend adhering to external standards I think that probably belongs under information governance